src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int FIGURE_COUNT = 100;
```

Comment:
FIGURE_COUNT? What Figure are you talking about? Think about the name (what is '100' in your app?)